DSharpPlus/Clients/DiscordClient.Events.cs

```diff
@@ -1,72 +1,79 @@
 ﻿using System;
-using Microsoft.Extensions.Logging;
 using DSharpPlus.EventArgs;
+using Emzi0767.Utilities;
+using Microsoft.Extensions.Logging;
 
 namespace DSharpPlus
 {
     public sealed partial class DiscordClient
     {
+        internal static TimeSpan EventExecutionLimit { get; } = TimeSpan.FromSeconds(1);
```

Comment:
I feel like there should be an option for this in the DiscordConfiguration.

Reply:
It won't be. The limit is soft, and it's there to inform the user they are stalling the queue.